src/main/resources/data.sql

```diff
@@ -32,8 +32,8 @@ insert into section(section_no, course_id, sec_id, term_id, building, room, time
 (6, 'cst338', 1, 9, '052', '100', 'M W 10:00-11:50', 'jgross@csumb.edu'),
 (7, 'cst338', 2, 9, '052', '100', 'M W 10:00-11:50', 'jgross@csumb.edu'),
 (8, 'cst363', 1, 9, '052', '104', 'M W 10:00-11:50', 'dwisneski@csumb.edu'),
-(9, 'cst363', 2, 9, '052', '102', 'M W 2:00-3:50', 'dwisneski@csumb.edu'),
-(10, 'cst438', 1, 9, '052', '222', 'T Th 12:00-1:50', 'dwisneski@csumb.edu');
+(9, 'cst363', 2, 10, '052', '102', 'M W 2:00-3:50', 'dwisneski@csumb.edu'),
+(10, 'cst438', 1, 10, '052', '222', 'T Th 12:00-1:50', 'dwisneski@csumb.edu');
```

Comment:
I know I asked you to remove my changes, but you can keep these in so that we are all consistent in case you use this data in any capacity for your tests.